src/state/notifications/selectors.ts

```diff
@@ -22,6 +22,12 @@ export const readableError = createSelector(notificationError, err => {
   if (err.startsWith(messages.requestFailed5XX)) {
     return i18n.connectionIssue.couldntConnectToServer;
   }
+  if (err === messages.alreadySubscribed) {
+    return messages.alreadySubscribedReadable;
```

Comment:
it should come from i18n

Reply:
no, because it is temporary, this error will not have a chance to show up when the backend is done. As we don't deploy the app with partial email notifications, I don't want to give more phrases to translators as needed.

Reply:
I guess it can come from the backend when for example someone will subscribe to the same mail and the same wallet at the same time from a different device

Reply:
then checkingSubscription would be suspected

Reply:
what about situation
1. We are on confirm screen
2. Someone makes an API call for example from the postman and subscribe with the same wallet and email
1. We enter a pin
3. We send a confirm message
4. We will get this error

Reply:
I think that at that moment session token would be dead, so I would rather expect a different error or silent error and pretending that everything is fine not to confuse the user more than it's necessary. 
For sure that's a case to cover, but rather when the planned backend changes are done.

Reply:
so why do we care about making this error readable? let's delete it, render raw  error, wait for the backend, and then cover all the cases